### ISSUE TITLE
We need a plugin for CKEditor

### ISSUE THREAD

**Issue by mgifford** [dequelabs/axe-core#1466]
I'm cross-posting from https://github.com/cksource/ckeditor-plugin-a11ychecker/issues/242 to hopefully have a new accessibility checker engine brought into the very popular CKEditor. 

Would love it if there were more in the aXe community who could help make this a reality. 

This is following a tweet from https://twitter.com/dequesystems/status/1112712257195180032

**Comment by straker** [comment]
Hey Mike,

I took a cursory look at this (I'm not very familiar with CKEditor codebase). I figured out how to get axe to run on the editor DOM, but after that point I'm not sure what would need to be done. 

Since axe will mostly only run on visible nodes, all I did was remove the `display: none` property on `contentElement.$` before running axe, then swapped out this line https://github.com/cksource/ckeditor-plugin-a11ychecker/blob/master/EngineQuail.js#L126 for 

```js
contentElement.$.style.display = 'block';
axe.run(contentElement.$, function(error, results) {
  contentElement.$.style.display = 'none';
  if (error) throw error;

  // can now look at any violations in `results.violations`
});
```

After that point, I'm not sure what would need to happen to have the editor look at each violation in turn and make suggestions as it currently does, but hopefully it gets someone started. Happy to answer any questions, possibly continue the conversation in the ckeditor-plugin-a11ychecker tread.
